src/Chainweb/Pact/PactService.hs

```diff
@@ -971,7 +971,7 @@ execValidateBlock currHeader plData = do
     isGenesisBlock = isGenesisBlockHeader currHeader
 
 validateSilenceDates :: MonadThrow m => BlockHeader -> PayloadData -> m ()
-validateSilenceDates bh plData = case txSilenceEndDate (_blockChainwebVersion bh) of
+validateSilenceDates bh plData = case txActivationDate (_blockChainwebVersion bh) of
```

Comment:
This is a change in a core validation rule. Merging this change is a hard fork.

Reply:
Don't change this rule! It's eternal.